Kombit.Samples.CH.WebsiteDemo/MyPage.aspx.cs

```diff
@@ -96,12 +96,15 @@ protected void Btn_Logoff_Click(object sender, EventArgs e)
         /// <returns></returns>
         protected static string RenderAttributeValue(string name, string value)
         {
-            if (name != "dk:gov:saml:attribute:Privileges_intermediate")
-                return value;
-            //Below is the code to parse bpp value and handle the role validation process
-            //var decodedBpp = Encoding.UTF8.GetString(Convert.FromBase64String(value));
-            var bppGroupsList = PrivilegeGroupParser.Parse(value);
-            return PrivilegeGroupParser.ToJsonString(bppGroupsList);
+            if (name == "dk:gov:saml:attribute:Privileges_intermediate" || name == "https://data.gov.dk/model/core/eid/privilegesIntermediate")
+            {
+                //Below is the code to parse bpp value and handle the role validation process
+                //var decodedBpp = Encoding.UTF8.GetString(Convert.FromBase64String(value));
+                var bppGroupsList = PrivilegeGroupParser.Parse(value);
+                return PrivilegeGroupParser.ToJsonString(bppGroupsList);
+            }
+            
+            return value;            
         }
 
         protected static bool ValidateKombitAttributeProfile(Saml20Identity current)
```

Comment:
I think code needs to say what claims are missing. It was quite annoying to get an error message without knowing exactly what was wrong with the old sample.

Reply:
Fixed, update error message to show what claim types are missing